packages/discordx/src/decorators/decorators/Bot.ts

```diff
@@ -13,20 +13,11 @@ import {
 
 /**
  * Execute your application command, button, select menu, simple command or event by defined bot when multiple bots are running in the same instance
- * @param botId id of your bot
```

Comment:
Why did you remove all the param descriptions?

Reply:
![image](https://user-images.githubusercontent.com/43092101/154045905-e2735885-181c-4f53-98d8-e1f843579fe7.png)

When we use our IDE, the options are already there, so it feel like duplicate descriptions.

Reply:
They will be back with improved version